api/views/assets.py

```diff
@@ -433,6 +433,32 @@ def post(self, request):
         return Response(data=response, status=200)
 
 
+class FileDownloads(APIView):
+    permission_classes = [IsAuthenticated, IsAdminUser]
+
+    def get(self, request):
+        email = request.user.email
+        query_dict = request.query_params.dict()
+        file_name = query_dict.get('filename')
+
+        if file_name == 'skipped_assets':
+            filename = "{}.csv".format(email.split("@")[0])
+
+        if file_name == 'sample':
+            filename = "sample_import.csv"
+
+        if file_name == 'asset_details':
+            filename = "{}_exported_assets.xlsx".format(email.split("@")[0])
+
+        # send file
+        file_path = os.path.join(settings.BASE_DIR, "files/{}".format(filename))
```

Comment:
we can pass the `filename` here without the checks from [L444](https://github.com/AndelaOSP/art-backend/pull/298/files#diff-3eb66745053f6b976b9e9ca1186bfd4cR444) to [L451](https://github.com/AndelaOSP/art-backend/pull/298/files#diff-3eb66745053f6b976b9e9ca1186bfd4cR451), then use try/except/else to raise an appropriate error when the file doesn't exist